lars-reimann/api-editor
# refactor: restructure server packages & include Simple-ML core Jar

Closes partially #327.

### Summary of Changes

* Restructure server packages (flatter hierarchy)
* Include Simple-ML core Jar
  * Use this to generate code for function stubs

### Testing instructions

* Structure should make sense, but no need to go through all the changes to names and imports
* Should still be able to generate code from the client
* Tests should still run

### Additional notes

* I'll also update the stub creators for classes and modules in a future PR.
* The Jar will be pulled from Maven Central once it's published there.
